### PR TITLE
Powheg fragments for semi-invisible Higgs decay

### PR DESCRIPTION
**Pull request by yiiyama**
Three fragments based on corresponding existing Higgs samples, just with BSM Higgs decay into a photon and a massless dark photon.

### CODE REVIEW

**Comment by yiiyama** [comment]
Analysis:
https://indico.cern.ch/event/705911/contributions/2897656/attachments/1602163/2540464/semi_invisible_H.pdf
Google doc for EXO MC:
https://docs.google.com/spreadsheets/d/1crWtQbytkrf0KLpBr7lE1lRuTCCfF1nw--mVnGZ5Zrg/edit?usp=sharing

**Comment by perrozzi** [comment]
why CUETP8M1 ? You should use CP5

**Comment by yiiyama** [comment]
These signal samples are for a 2016 analysis and will be compared against background MC done with CUETP8M1. I actually don't know how dramatic the difference between the tune is though - is stacking a signal with CP5 over CUETP8M1 background a legitimate thing to do?

**Comment by efeyazgan** [comment]
Yes, e.g. 2016 ttbar analysis used CUETP8M2T4 for signal and CUETP8M1 for backgrounds.

**Comment by yiiyama** [comment]
OK, I switched to CP5 and renamed the files accordingly.

File: python/ThirteenTeV/Higgs/Hadronizer_TuneCP5_13TeV_powhegEmissionVeto_1p_HToGDarkG_M-125_LHE_pythia8_cff.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            'POWHEG:nFinal = 1',   ## Number of final state particles
+                                   ## (BEFORE THE DECAYS) in the LHE
+                                   ## other than emitted extra parton
+            '3000022:all = DarkPhoton void 3 0 0 0. 0. 0. 0. 0.',
+            '3000022:isVisible = 0',
+            '25:m0 = 125.0',
+            '25:oneChannel = 1 1 100 22 3000022'
+          ),
+        parameterSets = cms.vstring('pythia8CommonSettings',
+                                    'pythia8CUEP8M1Settings',
```
Comment (perrozzi) [inline python/ThirteenTeV/Higgs/Hadronizer_TuneCP5_13TeV_powhegEmissionVeto_1p_HToGDarkG_M-125_LHE_pythia8_cff.py]: need to update this

Comment (yiiyama) [reply python/ThirteenTeV/Higgs/Hadronizer_TuneCP5_13TeV_powhegEmissionVeto_1p_HToGDarkG_M-125_LHE_pythia8_cff.py]: Good catch, thank you. Fixed in 8feae8b.

**Pull request merged**